3]. Competitive Programming/HackerRank/Problem Solving/Algorithms/Easy/A Very Big Sum.py

```diff
@@ -0,0 +1,6 @@
+
+# Contributed by Paraj Shah
```

Comment:
Check this code again

Reply:
![image](https://user-images.githubusercontent.com/65371747/122299666-ea7c1880-cf1b-11eb-8523-0dc16947b844.png)
![image](https://user-images.githubusercontent.com/65371747/122299707-f8ca3480-cf1b-11eb-9795-6a94b30a0730.png)

Reply:
![image](https://user-images.githubusercontent.com/65371747/122299926-3e86fd00-cf1c-11eb-90ba-469f996bd39f.png)